tests/test_jointcal_hsc.py

```diff
@@ -2,6 +2,9 @@
 
 from __future__ import division, absolute_import, print_function
 
+import matplotlib
```

Comment:
It was necessary to override the backend before something much further down in the stack messed it up. I've added a comment.